src/MongoDB/Driver/Manager.cpp

```diff
@@ -17,6 +17,7 @@
 #include "hphp/runtime/ext/extension.h"
 #include "hphp/runtime/vm/native-data.h"
 #include "hphp/runtime/base/array-iterator.h"
+#include "hphp/runtime/base/execution-context.h"
```

Comment:
Was this necessary for the new functionality or part of an HHVM compatibility update?